aspnet/Chart/Overview.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Chart | ASP.NET Webforms | Syncfusion
-description: overview
+title: Overview of ASP.NET Webforms Chart control | Syncfusion
```

Comment:
Change like this - ASP.NET Web Forms, check and change in all places.

Reply:
changed